src/Mvc/Mvc.RazorPages/src/Infrastructure/PageRequestDelegateFactory.cs

```diff
@@ -30,6 +30,20 @@ internal sealed class PageRequestDelegateFactory : IRequestDelegateFactory
         private readonly IActionContextAccessor _actionContextAccessor;
         private readonly bool _enableActionInvokers;
 
+        public PageRequestDelegateFactory(
```

Comment:
I did not find any existing tests that would cover this class or its constructors explicitly and since the change is only structural I figured that no new tests are required. Please let me know if that is incorrect.